### PR TITLE
Implemented Regional API URL in config instead of hardcoded

### PR DESCRIPTION
**Pull request by sluongng**
# What did you implement:

Addressed #31 and #29 

# How did you implement it:

- Created 2 new classes: Region and BaseUrl following the Key class template
- BaseUrl will use Region.get() and Region was exposed and totally configurable.
- Default region is set to 'westus' to avoid any disruption to existing code bases.
- Added REGION config template to `/cognitive_face/tests/config.sample.py`
- Fixed some existing typos

# How can we verify it:

I have modified the test suit appropriately.

Just run `python setup.py test`

# Todos:

 - [X] Write tests
 - [X] Write documentation
 - [X] Fix linting errors
 - [X] Make sure code coverage hasn't dropped
 - [X] Provide verification config/commands/resources
 - [X] Change ready for review message below
 

**Is this ready for review?:** YES

**Is it a breaking change?:** NO

### CODE REVIEW

File: cognitive_face/util.py [hunk 1]
```diff
@@ -11,7 +11,35 @@
...
+        """Get the API region"""
+        if not hasattr(cls, 'region'):
+            cls.region = 'uswest'
+        return cls.region
+
+
+class BaseUrl(object):
+
+    @classmethod
+    def set(cls, url):
```
Comment (huxuan) [inline cognitive_face/util.py]: Prefer to use `base_url` for consistency.

**Comment by huxuan** [comment]
Hi @sluongng, really appreciate for what you have done. Just some minor suggestions wish you can consider in your pull request.

**Comment by sluongng** [comment]
I have modified my solution as requested. Feedbacks are more than welcome.

**Comment by huxuan** [comment]
Thanks for your contribution!

**Pull request merged**